Source/WebCore/rendering/svg/RenderSVGInlineText.cpp

```diff
@@ -85,7 +85,7 @@ String RenderSVGInlineText::originalText() const
 
 void RenderSVGInlineText::setRenderedText(const String& text)
 {
-    RenderText::setRenderedText(text);
+    RenderText::setRenderedText(applySVGWhitespaceRules(text, style().whiteSpace() == WhiteSpace::Pre));
```

Comment:
I'm a little concerned about the perf impact. Does this always cause a string copy? Can we optimize it so that if no replacements are needed, there is no string copy?